raiden_contracts/data/source/raiden/TokenNetwork.sol

```diff
@@ -1095,9 +1136,9 @@ contract TokenNetwork is Utils, Controllable {
         pure
         returns (bytes32)
     {
-        require(participant != address(0x0));
-        require(partner != address(0x0));
-        require(participant != partner);
+        require(participant != address(0x0), "TN: participant address zero");
+        require(partner != address(0x0), "TN: partner address zero");
+        require(participant != partner, "TN: identical addresses");
```

Comment:
I note that these are identical to the ones in `getChannelIdentifier`, but I guess that should be fine?
Assuming one could determine where it went wrong from the context...

Reply:
Mh, good point. I guess it would be ok to leave it as is, but an unambiguous message would be nicer! WHat do you think about preficing these here with `TN/hash:`?

Reply:
I'm fine with either way, as I don't think it would matter as the rest of the messages, I just wanted to point it out in case you were not already aware.

Reply:
OK, then let's leave it like it is for now and if we run into problems it's easy to change.